src/battle/area/mim/actor/forest_fuzzy.c

```diff
@@ -426,11 +426,11 @@ EvtScript N(80219054) = {
         EVT_CALL(PlaySoundAtActor, ACTOR_SELF, SOUND_214)
         EVT_CALL(GetActorPos, ACTOR_SELF, LVar0, LVar1, LVar2)
         EVT_ADD(LVar1, 10)
-        EVT_CALL(N(UnkBattleFunc2), LVar0, LVar1, LVar2, LVar3)
+        EVT_CALL(N(StartEnemyDrainGFX), LVar0, LVar1, LVar2, LVar3)
```

Comment:
GFX should be avoided when not referring to Gfx (display list) data. Elsewhere in the codebase, we would name these functions SpawnXxxEffect or SpawnXxxFX. So something like SpawnDrainHealthStartFX and SpawnDrainHealthContinueFX would be consistent, albeit unwieldy.